STEERING-COMMITTEE.md

```diff
@@ -46,6 +49,40 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
```

Comment:
I see the tests as the somewhat mechanical validation of a specification, and reviewing changes to a specification feels like higher signal/noise than reviewing changes to a test suite.

Reply:
This implies that the trademark committee will have members that are actively engaged on PRs that touch the API and conformance suite, does it not? Reading this with a fresh mind, I'm reading this as trademark committee has to approve any such PR. Is that accurate?

Reply:
@pmorie Yes but potentially as someone suggested on release boundaries rather than on a PR by PR basis. I can see arguments for both approaches, we should choose an approach that's effective/efficient and minimizes any wasted work.

Reply:
What about simply delegating conformance issues to a “Conformance WG” which will be handled like other WGs?

Reply:
Do we happen to have an example of how this would work in practice?  Regardless of the approach (by PR basis or release boundaries? For example, a few upstream contributors want to form Functions WG (for example, could also be teacups or 'The Concern'), ToC approves this request.  The WG develops interesting and meaningful functionality, and want to be added to 'Core' (or whatever we end up calling it).  The WG submit a PR to the conformance suite, ToC approves of it, KSC approves of it. A member (any of the member orgs) of the KSC wants to VETO this PR.  Does it come in the form rejecting the PR?

Reply:
Or other example, from real life. Eventing has open up boundaries (no auth) to submit an event. If TOC/WG choose to patch that with a requirement of requiring an auth, is that a CORE change? Would that be veto by KSC?